### PR TITLE
Inserting gazebo msgs

### LINKED ISSUE #14: After pulling update no libForwardLookingGazeboSonar.so file found
```
[Err] [Plugin.hh:165] Failed to load plugin libForwardLookingGazeboSonar.so: libForwardLookingGazeboSonar.so: cannot open shared object file: No such file or directory
```
in `/catkin_ws/install/lib/` only `libForwardLookingSonarGazebo.so` and `libFowardLookingGazeboSonar.so` found.

### PR DESCRIPTION
**Pull request by ivan1993br**
Inserting sonar message communication for sonar
Close #14

### CODE REVIEW

**Comment by g-arjones** [comment]
You need to add sonar_msgs in the package set before merging this

File: src/FLSonar.cc [hunk 1]
```diff
@@ -602,6 +611,24 @@ void FLSonar::GenerateTransferTable(std::vector<int> &_transfer)
   }
 }
 
+//////////////////////////////////////////////////
+sonar_msgs::SonarStamped FLSonar::SonarRosMsg(gazebo::physics::WorldPtr _world)
```
Comment (matheusns) [inline src/FLSonar.cc]: I believe that the `gazebo::physics::WorldPtr` getter methods are constants, thus you could pass the pointer `world` as a `const` reference.

**Comment by ivan1993br** [comment]
> You need to add sonar_msgs in the package set before merging this

I have done a pull request on the common package with the sonar msgs

**Comment by g-arjones** [comment]
> I have done a pull request
I think you forgot

**Comment by ivan1993br** [comment]
> > I have done a pull request
> 
> I think you forgot

You were right, i have just done a [PR](https://github.com/Brazilian-Institute-of-Robotics/bir.ros-common_package_set/pull/19)

File: src/FLSonar.cc (hunk 1 above)
Comment (ivan1993br) [reply src/FLSonar.cc]: I agree, and I have done this modification

**Comment by ivan1993br** [comment]
I will close this PR

**Pull request merged**